source/whats-new.txt

```diff
@@ -33,6 +41,30 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.13:
+
+What's New in 1.13
+------------------
+
+The 1.13 {+driver-short+} release includes the following improvements and fixes:
+
+- Logging for server selection and SDAM. See the :ref:`golang-logging` documentation
+  for more information about logging.
+
+- Methods on the ``mongo.Collection`` type for managing search indexes programmatically.
+
+- The ``event.CommandStartedEvent`` and ``event.CommandFinishedEvent`` events return
+  the ``DatabaseName`` field. This field is also included in the command-logging analogs.
+
+- In a sharded topology, when selecting a server to retry a failing query, the driver
```

Comment:
vale fix
```suggestion
- In a sharded topology, when selecting a server to retry an unsuccesful query, the driver
```

Reply:
so silly to me that vale complains about this.

Reply:
@mongoKart 

Just to provide context, this is directly from [Negative Words](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/#avoid-negative-words) guidance in the style guide. I think it's reasonable guidance, but can understand that it's a subjective rule.

Here's my suggestion:
If you want to identify specific cases in which we should allow negative words, can you include those exceptions in a style guide PR? That way, other team reps can weigh in. Based on the results, we can update the Vale rule.
Thanks!